src/utils/getContractOrSystemName.ts

```diff
@@ -0,0 +1,31 @@
+import i18next from 'i18next';
+
+import { SystemIntake_systems as System } from 'queries/types/SystemIntake';
+import { RequestRelationType } from 'types/graphql-global-types';
+
+/** Returns linked contract or system name for display */
+const getSystemOrContractName = (
```

Comment:
This function name is in the opposite order as the file name. Might be worth aligning them if you think that might cause confusion later.

Reply:
Ah good catch, thanks for pointing this out! Just pushed changes to update the file name.